# RNA Sequencing


# Introduction

RNA-seq also known as RNA sequencing is a technique that by using next generation sequencing (NGS) can examine the transcriptome to identify which genes within our DNA are activated or deactivated, as well as the level of activity <sup>1</sup>. The transcriptome encompasses all the RNA transcripts in a cell at a specific developmental stage or condition, crucial for understanding genome function, cell composition, and disease mechanisms <sup>1</sup>. Transcriptomics aims to catalog various RNA species, determine gene transcription structures, and measure transcript expression changes across different states <sup>1</sup>.


# Technological Advancements in RNA Sequencing

Hybridization-based methods like microarrays are cost-effective but limited by background noise and reliance on prior genomic knowledge <sup>1</sup>. Sequence-based methods, including Sanger and tag-based approaches, offer greater precision but are expensive and face challenges in quantifying and mapping isoforms <sup>2–5</sup>. RNA-Seq, a recent advancement, overcomes many of these issues by providing detailed and comprehensive transcriptome analysis and offers precise mapping and quantification of transcripts, surpassing traditional methods in accuracy and scope <sup>1</sup>. It has been successfully applied to various organisms, including yeast, plants, and humans, and is poised to revolutionize the study of eukaryotic transcriptomes by providing a detailed and comprehensive view of gene expression <sup>6–12</sup>.


# The Role of RNA-Seq in Cancer Research

Single-cell RNA sequencing (scRNA-seq) is a technology that analyzes gene expression at the resolution of individual cells, providing detailed insights into cellular heterogeneity <sup>13</sup>. It has become a powerful tool in cancer research, enabling detailed exploration of cancer stem cells, tumor microenvironments, and therapy-induced changes <sup>14–16</sup>. It enhances our understanding of cellular interactions and spatial organization when combined with techniques like cell sorting and ligand-receptor interaction <sup>17,18</sup>. Innovations such as CRISPR-pooled screens, single-cell metabolically labeled RNA tagging (ScNT-seq), and MIX-Seq further refine our ability to analyze genomic perturbations, transcriptional activity at various stages, and cancer cell responses to treatments <sup>17–21</sup>. Additionally, single-nucleus RNA sequencing (snRNA-seq) aids in detecting cellular heterogeneity and specific cancer-related sub-populations <sup>22</sup>. Collectively, these advanced RNA-seq technologies, especially when integrated with spatial transcriptomics and multi-omics data, offer new insights into cancer diagnosis, treatment, and prevention <sup>23</sup>.


# Conclusion

RNA sequencing (RNA-Seq) has revolutionized transcriptome analysis by offering precise and comprehensive insights into gene expression <sup>1,6–12</sup>. Its evolution from traditional methods to advanced techniques like single-cell RNA sequencing (scRNA-seq) and its applications in cancer research have significantly enhanced our understanding of cellular heterogeneity, gene activity, and disease mechanisms <sup>13–22</sup>. By integrating RNA-Seq with spatial transcriptomics and multi-omics data, researchers are now better equipped to explore cancer diagnostics, treatment, and prevention, paving the way for more personalized and effective strategies in cancer management <sup>23</sup>.

 


# References

1.        Wang, Z., Gerstein, M. & Snyder, M. RNA-Seq: a revolutionary tool for transcriptomics. _Nat Rev Genet_ **10**, 57–63 (2009).

2.        Harbers, M. & Carninci, P. Tag-based approaches for transcriptome research and genome annotation. _Nat Methods_ **2**, 495–502 (2005).

3.        Gerhard, D. S. _et al._ The status, quality, and expansion of the NIH full-length cDNA project: the Mammalian Gene Collection (MGC). _Genome Res_ **14**, 2121–2127 (2004).

4.        Velculescu, V. E., Zhang, L., Vogelstein, B. & Kinzler, K. W. Serial analysis of gene expression. _Science (1979)_ **270**, 484–487 (1995).

5.        Boguski, M. S., Tolstoshev, C. M. & Bassett Jr, D. E. Gene discovery in dbEST. _Science (1979)_ **265**, 1993–1994 (1994).

6.        Nagalakshmi, U. _et al._ The transcriptional landscape of the yeast genome defined by RNA sequencing. _Science (1979)_ **320**, 1344–1349 (2008).

7.        Wilhelm, B. T. _et al._ Dynamic repertoire of a eukaryotic transcriptome surveyed at single-nucleotide resolution. _Nature_ **453**, 1239–1243 (2008).

8.        Mortazavi, A., Williams, B. A., McCue, K., Schaeffer, L. & Wold, B. Mapping and quantifying mammalian transcriptomes by RNA-Seq. _Nat Methods_ **5**, 621–628 (2008).

9.        Lister, R. _et al._ Highly integrated single-base resolution maps of the epigenome in Arabidopsis. _Cell_ **133**, 523–536 (2008).

10.      Cloonan, N. _et al._ Stem cell transcriptome profiling via massive-scale mRNA sequencing. _Nat Methods_ **5**, 613–619 (2008).

11.      Morin, R. D. _et al._ Profiling the HeLa S3 transcriptome using randomly primed cDNA and massively parallel short-read sequencing. _Biotechniques_ **45**, 81–94 (2008).

12.      Marioni, J. C., Mason, C. E., Mane, S. M., Stephens, M. & Gilad, Y. RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays. _Genome Res_ **18**, 1509–1517 (2008).

13.      Jovic, D. _et al._ Single‐cell RNA sequencing technologies and applications: A brief overview. _Clin Transl Med_ **12**, e694 (2022).

14.      Maynard, A. _et al._ Therapy-induced evolution of human lung cancer revealed by single-cell RNA sequencing. _Cell_ **182**, 1232–1251 (2020).

15.      Pan, X. _et al._ Identification of a novel cancer stem cell subpopulation that promotes progression of human fatal renal cell carcinoma by single-cell RNA-seq analysis. _Int J Biol Sci_ **16**, 3149 (2020).

16.      Li, Y.-L. _et al._ Single-cell analysis reveals immune modulation and metabolic switch in tumor-draining lymph nodes. _Oncoimmunology_ **9**, 1830513 (2020).

17.      Jaitin, D. A. _et al._ Dissecting immune circuits by linking CRISPR-pooled screens with single-cell RNA-seq. _Cell_ **167**, 1883–1896 (2016).

18.      Datlinger, P. _et al._ Pooled CRISPR screening with single-cell transcriptome readout. _Nat Methods_ **14**, 297–301 (2017).

19.      Genga, R. M. J. _et al._ Single-cell RNA-sequencing-based CRISPRi screening resolves molecular drivers of early human endoderm development. _Cell Rep_ **27**, 708–718 (2019).

20.      Qiu, Q. _et al._ Massively parallel and time-resolved RNA sequencing in single cells with scNT-seq. _Nat Methods_ **17**, 991–1001 (2020).

21.      McFarland, J. M. _et al._ Multiplexed single-cell transcriptional response profiling to define cancer vulnerabilities and therapeutic mechanism of action. _Nat Commun_ **11**, 4296 (2020).

22.      Sun, W. _et al._ snRNA-seq reveals a subpopulation of adipocytes that regulates thermogenesis. _Nature_ **587**, 98–102 (2020).

23.      Hong, M. _et al._ RNA sequencing: new technologies and applications in cancer research. _J Hematol Oncol_ **13**, 1–16 (2020).

 
